src/genie/libs/parser/iosxr/tests/ShowImDampening/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,41 @@
+expected_output = {
+                    'interface': {
+                        'GigabitEthernet0/0/0/0': {
+                            'protocol': {
+                                'not_present': {
+                                    'capsulation': 'not_present',
+                                    'penalty': 629,
+                                    'suppressed': 'no'
+                                }
+                            }
+                        },
+                        'GigabitEthernet0/0/0/1': {
+                            'protocol': {
+                                'not_present': {
+                                    'capsulation': 'not_present',
+                                    'penalty': 2389,
+                                    'suppressed': 'yes'
```

Comment:
take as is. `YES`

Reply:
Noted, all these comments will be corrected in the next update. Thanks.

Reply:
confirmed the change